data/frameReceiver/src/MercuryFrameDecoder.cpp

```diff
@@ -20,6 +20,7 @@
 using namespace FrameReceiver;
 
 const std::string MercuryFrameDecoder::CONFIG_FEM_PORT_MAP = "fem_port_map";
+const std::string MercuryFrameDecoder::CONFIG_PACKET_HEADER_EXTENDED = "packet_header_extended";

```

Comment:
To me this would make more sense (and be more readable in config files) as `extended_packet_header`